databricks/provider_test.go

```diff
@@ -216,10 +216,3 @@ func TestAccDatabricksCliConfigWorks(t *testing.T) {
 		},
 	)
 }
-
-// getAndAssertEnv fetches the env for testing and also asserts that the env value is not Zero i.e ""
-func getAndAssertEnv(t *testing.T, key string) string {
-	value, present := os.LookupEnv(key)
-	assert.True(t, present, fmt.Sprintf("Env variable %s is not set", key))
-	return value
-}
```

Comment:
I think we should have this as a way to test the env variable is set so you get an error in the stack saying that the expected env variable is not set

Reply:
tests should be skipped if env variable is not set. e.g. in very big run of all tests suddenly env is not there. e.g. AWS/Azure combinations. test should decide either or not it should be run, depending on the environment.

Reply:
I am referring to when you have CLOUD_ENV set to azure but the but the service principal env variables are not there. We should make sure we catch them and let the user know why the test was skipped. We do not have to run the assertion but i should know that i forgot to set an env variable thats why the test was skipped.